packages/frontend/src/pages/insight-editor/components/insight-editor-sidebar/sidebar-files.tsx

```diff
@@ -183,14 +187,34 @@ export const SidebarFiles = ({
     md: isFilesOpen ? iconFactoryAs('chevronRight') : iconFactoryAs('folderOpened')
   });
 
+  const {
+    getInputProps,
+    getRootProps,
+    isDragActive,
+    open: openFilePicker
+  } = useDropzone({
+    onDrop: (acceptedFiles) => onDropFile(acceptedFiles, undefined),
+    noClick: true
+  });
+
+  const dragBgColor = useColorModeValue('snowstorm.200', 'polar.100');
+  const dragBorderColor = useColorModeValue('polar.200', 'snowstorm.300');
+
   return (
     <Flex
       direction="column"
       align="stretch"
-      flexBasis={{ base: 'unset', md: '20rem', xl: '22rem' }}
-      maxW={{ base: 'unset', md: '20rem', xl: '26rem' }}
+      flexGrow={1}
+      {...(isDragActive && {
+        bg: dragBgColor,
+        borderWidth: '1px',
+        borderStyle: 'dashed',
+        borderColor: dragBorderColor
+      })}
       {...flexProps}
+      {...getRootProps()}
     >
+      <input {...getInputProps()} />
```

Comment:
Nice!
Quick question, does this mean that even with the files collapsed the drag file action will work?

Reply:
Correct!

![Screen Shot 2022-04-12 at 3 44 59 PM](https://user-images.githubusercontent.com/3084806/163041422-793a14fe-7db1-4db3-bf2f-26793a518e49.png)

Reply:
Sweet! 😄